framegear/components/Frame/Frame.tsx

```diff
@@ -76,10 +85,6 @@ function ValidFrame({ metadata }: { metadata: FrameMetadataWithImageObject }) {
 }
 
 function ErrorFrame() {
-  // TODO: implement -- decide how to handle
```

Comment:
Definitely acknowledge that this isn't elegant, but let's not remove. We will work on cleaning up these comments.